src/Symfony/Component/HttpKernel/DataCollector/LoggerDataCollector.php

```diff
@@ -113,19 +113,28 @@ private function getContainerDeprecationLogs()
             return array();
         }
 
-        $stubs = array();
         $bootTime = filemtime($file);
+        $traceReflector = new \ReflectionProperty(\Exception::class, 'trace');
+        $traceReflector->setAccessible(true);
         $logs = array();
         foreach (unserialize(file_get_contents($file)) as $log) {
-            $log['context'] = array('exception' => new SilencedErrorContext($log['type'], $log['file'], $log['line']));
```

Comment:
I hesitated. I finally opt for the `\ErrorException` as we already get for runtime deprecations. 

Also `SilencedErrorContext` implements `JsonSerializable`; I wasn't convinced what to do with the trace regarding this. If it's cleaned from any object (by not having objects and arguments in the trace) its ok. But by adding a `$trace` argument, we can't assume the trace is safe and would need to be cleaned...or simply ignored in the serialized version.